recipe/meta.yaml

```diff
@@ -12,13 +12,14 @@ source:
 build:
   number: 0
   script:
-    - pip install --no-deps .
+    - python setup.py install
     - python -m ipykernel install --prefix $PREFIX  # [not win]
     - python -m ipykernel install --prefix %PREFIX%  # [win]
 
 requirements:
   build:
     - python
+    - setuptools
```

Comment:
Adding setuptools shouldn't be needed, and if it is, this change shouldn't be made.